pennylane/math/matrix_manipulation.py

```diff
@@ -140,6 +145,47 @@ def expand_matrix(base_matrix, wires, wire_order=None, sparse_format="csr"):
     return mat
 
 
+def reduce_operators(
+    ops: Sequence[np.ndarray], reduce_func: callable, sparse=False
```

Comment:
```suggestion
    ops: Sequence[Operator], reduce_func: callable, sparse=False
```

Reply:
I wonder if we could instead just accept the `mats_and_wires` gen.  Then we could remove the `sparse` keyword argument, and then the `math` module wouldn't depend on the `Operator` interface.

Reply:
I added the `mats_and_wires` inside the function because if not we would have the line repeated 4 times.

However it makes sense that the `math` module should be independent of the `Operator` interface. I will implement this change.

Reply:
Done!